packages/features/users/components/UserTable/UserTableActions.tsx

```diff
@@ -0,0 +1,176 @@
+import { ExternalLink, MoreHorizontal, Edit2, UserX, Lock } from "lucide-react";
+import type { Action } from "users/components/UserTable/UserListTable";
+
+import { classNames } from "@calcom/lib";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import {
+  ButtonGroup,
+  Tooltip,
+  Button,
+  Dropdown,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownItem,
+  DropdownMenuSeparator,
+} from "@calcom/ui";
+
+import type { User } from "./UserListTable";
+
+export function TableActions({
```

Comment:
Not much has changed from the Team way of handling this - just extracted to a new component.

Will cleanup old files once i have replaced team list with the new table